charts/templates/cluster-global-sidecar.yaml

```diff
@@ -95,6 +105,11 @@ spec:
       serviceAccountName: global-sidecar
       containers:
         - name: global-sidecar
+          {{- if $f.disasterRecoveryPorts }}
```

Comment:
`disasterRecovery` is a scenario or an use case but not a feature. we should make it clear.